ui-v2/app/components/tabular-collection.js

```diff
@@ -37,6 +56,21 @@ const change = function(e) {
     const value = e.currentTarget.value;
     if (value != get(this, 'checked')) {
       set(this, 'checked', value);
+      if (e.currentTarget.getAttribute('id') !== 'actions_close') {
+        const $tr = closest('tr', e.currentTarget);
+        const $group = [...$('~ ul', e.currentTarget)][0];
+        const $footer = [...$$('footer[role="contentinfo"]')][0];
+        const groupRect = $group.getBoundingClientRect();
+        const footerRect = $footer.getBoundingClientRect();
+        const groupBottom = groupRect.top + $group.clientHeight;
+        const footerTop = footerRect.top;
+        if (groupBottom > footerTop) {
+          $group.classList.add('above');
+        } else {
+          $group.classList.remove('above');
+        }
+        $tr.style.zIndex = maxZIndex + 1;
+      }
```

Comment:
When does this `change` fn fire?

Reply:
When a dropdown is opened (under the hood, this is when a radiobutton is selected).